SuperTiled2Unity/Assets/SuperTiled2Unity/Scripts/Editor/Loaders/TilesetLoader.cs

```diff
@@ -291,7 +291,11 @@ private void ProcessObjectGroupElement(SuperTile tile, XElement xObjectGroup)
 
                     collision.m_ObjectId = xObject.GetAttributeAs("id", 0);
                     collision.m_ObjectName = xObject.GetAttributeAs("name", string.Format("Object_{0}", collision.m_ObjectId));
-                    collision.m_ObjectType = xObject.GetAttributeAs("type", "");
+                    collision.m_ObjectType = xObject.GetAttributeAs("class", "");
+                    //As of Tiled 1.9 types have been merged with classes
+                    //As a simple way to support both version we can fall back like this to the old < 1.9 way
+                    if (string.IsNullOrWhiteSpace(collision.m_ObjectName))
```

Comment:
Probably you meant to check `m_ObjectType`.

```suggestion
                    if (string.IsNullOrWhiteSpace(collision.m_ObjectType))
```

For the record, I've undone this compatibility breakage in Tiled 1.10, but this is still a good change for those who are using Tiled 1.9 or those who have picked the "Tiled 1.9" compatibility mode in later versions.

Reply:
🤦